nilearn/image/resampling.py

```diff
@@ -520,20 +507,6 @@ def resample_img(img, target_affine=None, target_shape=None,
     else:
         resampled_data_dtype = data.dtype
 
-    if LooseVersion(scipy.__version__) >= LooseVersion('0.17'):
```

Comment:
I am not sure this should be removed ?

Reply:
It will always evaluate to True, now that SciPy is minimum v0.17. The logic will always be executed. No need for a condition check.

Reply:
I had accidentally removed the logic as well, not just the condition check which was causing the problem. Fixed. Thanks.